test/arrays/ferguson/from-iterator/for-yields-no-arrays.future

```diff
@@ -0,0 +1,2 @@
+design: runtime type missing for yield-nothing
+
```

Comment:
Need some more explanation here. (I had to figure it out myself).
There should probably be a github issue for this as well,
or just cross-reference with #8152.

Reply:
I made issue  #8156 for it, I just forgot to put it in the .future. I've fixed that now, thanks for pointing it out!